recipes/libedit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libedit" %}
+{% set version = "3.1" %}
+{% set date = "20170329" %}
+{% set sha256 = "91f2d90fbd2a048ff6dad7131d9a39e690fd8a8fd982a353f1333dd4017dd4be" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ date }}-{{ version }}.tar.gz
+  url: http://thrysoee.dk/editline/{{ name }}-{{ date }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Also-look-in-libtinfo-for-tgetent.patch
+
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - automake
+    - autoconf
+    - m4
+    - perl
```

Comment:
You need to pin perl

Reply:
Ok, thanks, to what though?  https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py doesn't list a pinning

Reply:
`5.22.0.1`. see https://github.com/conda-forge/conda-smithy/issues/415

Reply:
Ok! Thanks!

Reply:
That worked!